bakery/pipeline.yml

```diff
@@ -22,7 +22,7 @@ resources:
       user_access_token: ((slack-user-token))
       bot_access_token: ((slack-bot-token))
       bot_user_id: ((slack-bot-user-id))
-      regexp: '^<@((slack-bot-user-id))>\s+pdf-spike\s+(\S+)\s+(\S+)\s*$'
+      rregexp: '^<@((slack-bot-user-id))>\s+pdf-spike\s+(\S+)\s+(\S+)\s*$'
```

Comment:
Is this a typo?

Reply:
@karenc yes it was, it's been corrected.